### ISSUE TITLE
Learner view: searching for independent practice quiz

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8511]
### User stories for this issue:

> As a learner, I want to be able to search for an independent practice quiz in a channel so that I can find the right independent practice quiz.

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-20 at 9 32 43 PM](https://user-images.githubusercontent.com/13563002/138212023-7115c128-7989-467d-8e24-3369cf75f411.png)|Filtering by "Practice" activity returns both quizzes and exercises|
|![Screen Shot 2021-10-25 at 10.27.30 PM.png](https://images.zenhubusercontent.com/6050ea86e1747f15de7d8759/34d3ad84-f413-48db-aee2-92c494ac17e3)|Info panel is opened|

### Notes and changes to current designs:
- [Figma for search](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=916%3A6159)
- Filtering by 'practice' should return both exercises and quizzes
- All strings required for this component should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`
- There should be different behaviors depending on whether there is a single copy of the independent practice quiz or whether there are several copies in, for example, many lessons or many channels (we'll have to discuss this in more detail - please put in channel)

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**

_This issue is done when the Gherkin stories below work_

- [ ] Independent practice quizzes are discoverable in search results
```Feature: Independent practice quizzes are discoverable in search results
	
	Scenario: Independent practice quizzes showing up in search results
		Given that I am on the search page
			And I have made a search related to independent practice quizzes
		Then I see practice content types appear in the search results
	
	Scenario: clicking on an independent practice quiz
		Given that there are practice content types
		When I click on an independent practice quiz search result
			And it does not have copies in other tree locations
		Then I am on the channel location of that independent practice quiz

	Scenario: viewing the multiple locations of an independent practice quiz
		Given I am on the search page
			And there are independent practice quiz search results showing
			And there is an independent practice quiz with multiple locations, conveyed by an 'n locations' indicator on the content card
		When I click on that content card
		Then a dialog appears showing me the different locations of the independent practice quiz
```

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8511?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by bjester** [comment]
We will need to change how completion data is fetched for completed items to include the actual score

**Comment by bjester** [comment]
Separated out "Score: %%%" portion of feature into separate issue since it will require significant effort

**Comment by rtibbles** [comment]
Fixed in #8645
